adapters/sharethrough/sharethrough.go

```diff
@@ -1,62 +1,103 @@
 package sharethrough
 
 import (
+	"encoding/json"
 	"fmt"
-	"net/http"
-	"regexp"
-
 	"github.com/mxmCherry/openrtb/v15/openrtb2"
 	"github.com/prebid/prebid-server/adapters"
 	"github.com/prebid/prebid-server/config"
 	"github.com/prebid/prebid-server/errortypes"
 	"github.com/prebid/prebid-server/openrtb_ext"
+	"github.com/prebid/prebid-server/version"
+	"net/http"
+	"strings"
 )
 
-const supplyId = "FGMrCMMc"
-const strVersion = 8
+var strVersion = "10.0"
```

Comment:
I used a `var` to overwrite in tests, so it can be hardcoded there and we don't have to change all json files when the version is incremented, if I can do that with a const I'm cool with it.

I tried renaming to `version` but it conflicts with `github.com/prebid/prebid-server/version`, I'm going with `adapterVersion`

Reply:
> I used a `var` to overwrite in tests, so it can be hardcoded there and we don't have to change all json files when the version is incremented, if I can do that with a const I'm cool with it.

Ah, ok. That's fine.

> I tried renaming to `version` but it conflicts with `github.com/prebid/prebid-server/version`, I'm going with `adapterVersion`

Sounds good. Thank you.